### DISCUSSION TITLE
救命

### DISCUSSION THREAD

**Discussion by Fackcolo** [share121/inter-knot#942] [其他]
暑假作业要写不完了啊啊啊啊啊:（
有没有能把我作业送到空洞里的绳匠，放的地方越偏僻越好……

**Comment by WLMRMZ** [comment]
![image](https://github.com/user-attachments/assets/9cf1f73e-87ed-4dbb-a1db-5e26fda6a0a3)

**Comment by WLMRMZ** [comment]
我有个绳匠朋友来帮你了！
